x11-wm/e16/e16-1.0.30.ebuild

```diff
@@ -0,0 +1,146 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Enlightenment Window Manager (E16)"
+HOMEPAGE="https://www.enlightenment.org https://sourceforge.net/projects/enlightenment/"
+SRC_URI="https://downloads.sourceforge.net/projects/enlightenment/${P}.tar.xz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="amd64 x86"
```

Comment:
We drop keywords to unstable on version bumps and stabilize the package later.  Please run `ekeyword ~all name.ebuild`